src/endpoint.h

```diff
@@ -194,7 +199,7 @@ class UartEndpoint : public Endpoint {
 
     bool setup(UartEndpointConfig config); ///< open UART device and apply config
 
-    static const ConfFile::OptionsTable option_table[4];
+    static const ConfFile::OptionsTable option_table[5];
```

Comment:
eventually we need to get rid of this number here

Reply:
Using constexpr doesn't work (at least before C++17) and just removing the number breaks any sizeof() call. So I don't like that number either, but haven't found a solution yet.